tools/hangar/__snapshots__/tree_json.ts.snap

```diff
@@ -10,8 +10,8 @@ exports[`tree.json for an app with many resources 1`] = `
             "children": {
               "Code": {
                 "constructInfo": {
-                  "fqn": "@cdktf/provider-aws.s3Bucket.S3Bucket",
-                  "version": "12.0.2",
+                  "fqn": "cdktf.TerraformResource",
+                  "version": "0.15.2",
                 },
```

Comment:
Looks like we lost some information here (the tree.json no longer shows concrete information about specific AWS resources). This is a side effect of how we're no longer using the `@cdktf/provider-aws` package. It's not great, but I'm not sure how we can avoid it with our solution. (The alternative might be to "dynamically" install these dependencies when the user tries compiling their Wing app to `tf-aws` for the first time, or breaking up the SDK into multiple packages like @eladb was investigating).

Reply:
Is there a way to inject `terraformResourceType` into `tree.json` instead?

Reply:
#2347